grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -76,9 +79,11 @@ class JsonController {
         def requisition = Requisition.get(params?.requisition?.id)
         def product = Product.get(params?.product?.id);
         if (!requisition) {
+            response.status = 401
```

Comment:
Can you explain why this is 401 Unauthorized instead of 404 Not Found?

Reply:
ah, I wanted to use 400 - bad request, I'm not sure why I added 401. I will change it.